src/components/ratings_reviews/ReviewModal.jsx

```diff
@@ -0,0 +1,132 @@
+        <p>4 = Good</p>
+        <p>3 = Average</p>
+        <p>2 = Not terrible</p>
+        <p>1 = Bad</p>
+      </div>
+      <div className="do-you-recommend">
+        <p>Do you recommend this product?</p>
+        <input
+          type="radio"
+          value="Yes"
+          name="recommend-btn"
+          className="recommendation-radio-btns"
+        />
+        Yes
+        <input
+          type="radio"
+          value="No"
+          name="recommend-btn"
+          className="recommendation-radio-btns"
+        />
+        No
+      </div>
+      <div className="characteristics-radio-btns">
+        <div className="size-char">
+          <p className="char-words">Size</p>
+          <input type="radio" value="1" name="size" />
+          <input type="radio" value="2" name="size" />
+          <input type="radio" value="3" name="size" />
+          <input type="radio" value="4" name="size" />
+          <input type="radio" value="5" name="size" />
+        </div>
+        <div className="width-char">
+          <p className="char-words">Width</p>
+          <input type="radio" value="1" name="width" />
+          <input type="radio" value="2" name="width" />
+          <input type="radio" value="3" name="width" />
+          <input type="radio" value="4" name="width" />
+          <input type="radio" value="5" name="width" />
+        </div>
+        <div className="comfort-char">
+          <p className="char-words">Comfort</p>
+          <input type="radio" value="1" name="comfort" />
+          <input type="radio" value="2" name="comfort" />
+          <input type="radio" value="3" name="comfort" />
+          <input type="radio" value="4" name="comfort" />
+          <input type="radio" value="5" name="comfort" />
+        </div>
+        <div className="quality-char">
+          <p className="char-words">Quality</p>
+          <input type="radio" value="1" name="quality" />
+          <input type="radio" value="2" name="quality" />
+          <input type="radio" value="3" name="quality" />
+          <input type="radio" value="4" name="quality" />
+          <input type="radio" value="5" name="quality" />
+        </div>
+        <div className="length-char">
+          <p className="char-words">Length</p>
+          <input type="radio" value="1" name="length" />
+          <input type="radio" value="2" name="length" />
+          <input type="radio" value="3" name="length" />
+          <input type="radio" value="4" name="length" />
+          <input type="radio" value="5" name="length" />
+        </div>
+        <div className="function-char">
+          <p className="char-words">Function</p>
+          <input type="radio" value="1" name="function" />
+          <input type="radio" value="2" name="function" />
+          <input type="radio" value="3" name="function" />
+          <input type="radio" value="4" name="function" />
+          <input type="radio" value="5" name="function" />
+        </div>
+      </div>
```

Comment:
To save lines of code, next PR maybe try exploring creating a radio component where who you pass the name as a prop